app/pages/sections/call-off-ordering-party/ui.test.js

```diff
@@ -247,3 +256,67 @@ test('should navigate to task list page if save button is clicked and data is va
     .click(saveButton)
     .expect(getLocation()).eql('http://localhost:1234/order/organisation/order-id');
 });
+
+test('should show the error summary when there are validation errors', async (t) => {
+  nock(orderApiUrl)
+    .put('/api/v1/orders/order-id/sections/ordering-party')
+    .reply(400, putOrderingPartyErrorResponse);
+
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .click(saveButton);
+
+  await t
+    .expect(errorSummary.exists).ok()
+    .expect(errorSummary.find('li a').count).eql(1)
+    .expect(await extractInnerText(errorSummary.find('li a').nth(0))).eql('Telephone number must be 35 characters or fewer');
+});
+
+test('should show text fields as errors with error message when there are validation errors', async (t) => {
+  nock(orderApiUrl)
+    .put('/api/v1/orders/order-id/sections/ordering-party')
+    .reply(400, putOrderingPartyErrorResponse);
+
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const descriptionPage = Selector('[data-test-id="call-off-ordering-party-page"]');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const descriptionField = descriptionPage.find('[data-test-id="question-telephoneNumber"]');
+
+  await t
+    .expect(descriptionField.find('[data-test-id="textarea-field-error"]').exists).notOk()
```

Comment:
this should be a textfield that is should be trying to find right?

Reply:
good spot